metrics.tf

```diff
@@ -135,15 +144,27 @@ resource "observe_link" "metrics" {
   fields    = each.value.fields
   label     = each.key
 
-  for_each = var.enable_nginx_ingress_metrics ? merge({
-    "Ingress" = {
-      target = var.kubernetes.ingress.oid
-      fields = ["ingress:name", "exported_namespace:namespace", "clusterUid"]
-    },
-    "Pod" = {
-      target = var.kubernetes.pod.oid
-      fields = ["controller_pod:name", "controller_namespace:namespace", "clusterUid"]
-    }
-    }
-  ) : {}
+  for_each = merge(
+    var.enable_nginx_ingress_metrics ? {
+      "Ingress" = {
+        target = var.kubernetes.ingress.oid
+        fields = ["ingress:name", "namespace", "clusterUid"]
+      },
+      "Pod" = {
+        target = var.kubernetes.pod.oid
+        fields = ["pod:name", "namespace", "clusterUid"]
+      }
+    } : {},
+    var.nginx_plus && var.enable_nginx_ingress_metrics ? {
+      "Upstream Pod" = {
+        target = var.kubernetes.pod.oid
+        # fields = ["upstream_pod:name", "upstream_namespace:namespace", "clusterUid"]
```

Comment:
Do we need this comment in here anymore?

Reply:
lols no, thanks for flagging